translations/client_fr.ts

```diff
@@ -5981,7 +5981,7 @@ Le serveur a répondu avec l&apos;erreur : %2</translation>
     <message>
         <location filename="../src/gui/wizard/welcomepage.cpp" line="70"/>
         <source>Easy-to-use web mail, calendaring &amp; contacts</source>
-        <translation>Courrier électronique, calendrier et contacts en ligne faciles à utiliser</translation>
+        <translation>E-mail, agenda et contacts en ligne faciles à utiliser</translation>
```

Comment:
I plan to add a few more lines of code so that the general user gets the default folder as usual. Only in special cases do I want to force the user to manually select a folder without a default (or an institution-specific default).